webapp/src/blocks.tsx

```diff
@@ -449,6 +449,11 @@ export class Editor extends toolboxeditor.ToolboxEditor {
                     // }
                 }
             }
+
+            // reset tutorial hint animation on any blockly event
+            if (this.parent.state.tutorialOptions != undefined) {
+                this.parent.setTutorialHintAnimation();
```

Comment:
can we use something a bit more generic like "pokeUserActivity()" ? We should add it to download, menu items and somewhere in monaco where save happens (for python tutorials)